sample_factory/algo/runners/runner.py

```diff
@@ -541,6 +551,10 @@ def _check_heartbeat(self):
             log.error("Stopping training from lack of heartbeats from " + ", ".join(type_list))
             self._stop_training()
 
+        log.debug("Checking Queue Size")
```

Comment:
I think this would spam the logs too much.
Let's:
- remove "Checking queue size" log
- for the other log let's print using debug_log_every_n function with n=1000
- is process id a number here? It should be a unique name of the process or event loop. You can get it from EventLoop or EventLoopProcess class